.evergreen/run-orchestration.sh

```diff
@@ -63,11 +59,24 @@ elif virtualenv --system-site-packages venv || python -m virtualenv --system-sit
 fi
 cd -
 
-case "$OS" in
-  cygwin*)
-    ORCHESTRATION_ARGUMENTS="$ORCHESTRATION_ARGUMENTS -s wsgiref"
-    ;;
-esac
+ORCHESTRATION_ARGUMENTS="-e default -f $MONGO_ORCHESTRATION_HOME/orchestration.config --socket-timeout-ms=60000 --bind=127.0.0.1 --enable-majority-read-concern"
+if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
```

Comment:
It's always set in cygwin. It seems to be the standard way to check for windows: https://github.com/mongodb-labs/drivers-evergreen-tools/pull/2#discussion_r90750514